packages/task/src/node/task-server.slow-spec.ts

```diff
@@ -125,6 +124,17 @@ describe('Task server / back-end', function () {
         await p;
     });
 
+    it('task is executed successfully with cwd as a file URI', async function () {
+        const command = isWindows ? commandShortrunningindows : commandShortRunning;
+        const config = createProcessTaskConfig('shell', FileUri.fsPath(command), []);
+        (config as any).cwd = FileUri.create(wsRoot).toString();
```

Comment:
This doesn't build for me:

    lerna ERR! [lint] ERROR: /home/emaisin/src/theia/packages/task/src/node/task-server.slow-spec.ts:130:20 - Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.


I suggest adding an optional `cwd` parameter to `createProcessTaskConfig`.